tests/ssl_test.py

```diff
@@ -0,0 +1,28 @@
+from unittest import mock
+from unittest.mock import MagicMock
+
+import pytest
+import ssl
+from aiomcache import Client
+from .conftest import McacheParams
+
+
+@pytest.mark.asyncio
+async def test_ssl_params_forwarded_from_client() -> None:
+        client = Client("host", port=11211, ssl=True, ssl_handshake_timeout=20)
+
+        with mock.patch("asyncio.open_connection", return_value=(MagicMock, MagicMock)) as oc:
```

Comment:
Didn't get autospec to work here, but as this is a stdlib method, I don't expect it to change that much. I changed the test so that the mock returns autospecs created from `Stream(Reader|Writer)` to try to make it a little more robust.

Reply:
Well, the other concern is the classic `m.assret_called()` and similar typos. I've actually found tests in aiohttp which have literally been testing nothing for years, but would have been caught if we enforced autospec or similar on all mocks.

I'll have a quick look later, otherwise we'll just make do with this.

Reply:
Ah, yeah, that's a good point. I was mostly thinking about the return value. I've added `autospec` now, so that any misspelling would case a AttributeError.